adaptors/mqtt/pkg/physical/device.go

```diff
@@ -27,7 +27,7 @@ type Device interface {
 
 // NewDevice creates a Device.
 func NewDevice(log logr.Logger, meta metav1.ObjectMeta, toLimb MQTTDeviceLimbSyncer) Device {
-	log.Info("Created")
+	log.Info("Created ")
```

Comment:
remove empty space?

Reply:
This is to align the `shutdown` log content.